lib/experimental/Widgets/Content/CopyLabel/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { Check, LayersFront } from "@/icons"
+import { cn } from "@/lib/utils"
+import { forwardRef, useEffect, useState } from "react"
+interface CopyLabelType {
+  text: string
+}
+
+const COPIED_SHOWN_MS = 1250
+
+export const CopyLabel = forwardRef<HTMLDivElement, CopyLabelType>(
+  ({ text }, ref) => {
+    const [copied, setCopied] = useState(false)
+
+    useEffect(() => {
+      if (copied) {
+        const timer = setTimeout(() => setCopied(false), COPIED_SHOWN_MS)
+
+        return () => clearTimeout(timer)
+      }
+    }, [copied])
+
+    const copyHandler = async () => {
+      try {
+        await navigator.clipboard.writeText(text)
+        setCopied(true)
+      } catch (error) {
+        void error
+      }
+    }
+
+    return (
+      <div
+        ref={ref}
+        className={cn(
+          "group relative flex h-8 items-center justify-between rounded-md p-1.5 font-medium text-f1-foreground-secondary transition-colors duration-300 hover:cursor-pointer",
+          copied
+            ? "hover:bg-f1-background-positive"
+            : "hover:bg-f1-background-secondary"
+        )}
+        onClick={copyHandler}
+        role="button"
+        aria-label={`Copy ${text} to clipboard`}
```

Comment:
Hey @RamProg !! Just curious 👀 Shouldn't the `aria-label` text come from translations?

Reply:
great catch Miriam! We are exploring a unified approach to translations in the design system. Until it is done, we use English to keep things moving